<launch>
    <!--############### Node Configurations ##################-->
    <arg name="output" default="screen"/>
    <arg name="fps"    default="40"/>

    <!-- Camera Configurations -->
    <arg name="cam0" default="15508253"/>

    <node pkg="exposure_control" type="cam_test_node" name="cam_test" output="$(arg output)">
        <param name="fps" type="double" value="$(arg fps)"/>

        <param name="cam0/identifier" type="string" value="$(arg cam0)"/>
        <param name="cam0/camera_name" type="string" value="pg_$(arg cam0)"/>
        <param name="cam0/calib_url" type="string" value="file://${ROS_HOME}/camera_info/pg_$(arg cam0).yaml"/>

        <param name="auto_shutter" type="bool" value="false"/>
        <param name="shutter_ms" type="double" value="6"/>
        <param name="auto_gain" type="bool" value="false"/>
        <param name="gain_db" type="double" value="0"/>
	<!-- trigger: -1 = free running, 0 = external or software trigger -->
	<param name="trigger_source" type="int" value="0"/>
	<param name="exposure" type="bool" value="false"/>
	<param name="auto_exposure" type="bool" value="false"/>
	<remap from="cam_test/exposure" to="/exposure_control/exposure"/>

    </node>
</launch>
